hefquin-vocabs/feddesc.ttl

```diff
@@ -0,0 +1,90 @@
+@prefix rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs:   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix owl:    <http://www.w3.org/2002/07/owl#> .
+@prefix xsd:    <http://www.w3.org/2001/XMLSchema#> .
+@prefix dc:     <http://purl.org/dc/elements/1.1/> .
+@prefix fd:     <http://w3id.org/hefquin/feddesc#> .
+
+<http://w3id.org/hefquin/feddesc>
+    rdf:type owl:Ontology ;
+    dc:creator       "Olaf Hartig",
+                     "Robin Keskisärkkä" ;
+    dc:date          "2025-05-05" ;
+    dc:format        "RDF" ;
+    dc:identifier    <http://w3id.org/hefquin/feddesc> ;
+    dc:title         "Federation Description Vocabulary for HeFQUIN"@en ;
+    rdfs:label       "Federation Description Vocabulary for HeFQUIN"@en ;
+    dc:description   "An RDF vocabulary for describing members of a federation to be considered for querying by the HeFQUIN engine."@en ;
+    rdfs:comment     "An RDF vocabulary for describing members of a federation to be considered for querying by the HeFQUIN engine."@en .
+
+### ===== CLASS DECLARATIONS =====
+
+fd:FederationMember a owl:Class ;
+    rdfs:label "Federation Member"@en ;
+    rdfs:comment "An entity that is part of the federation and provides data via a specific interface."@en .
+
+fd:EndpointInterface a owl:Class ;
+    rdfs:label "Endpoint Interface"@en ;
+    rdfs:comment "An interface to access a federation member."@en .
+
+fd:SPARQLEndpointInterface a owl:Class ;
+    rdfs:subClassOf fd:EndpointInterface ;
+    rdfs:label "SPARQL Endpoint Interface"@en ;
+    rdfs:comment "An interface to access a federation member via the SPARQL protocol."@en .
+
+fd:FragmentInterface a owl:Class ;
+    rdfs:subClassOf fd:EndpointInterface ;
+    rdfs:label "Fragment Interface"@en ;
+    rdfs:comment "An interface to access a federation member via pattern fragments, such as Triple Pattern Fragments or Bindings-Restricted Triple Pattern Fragments."@en .
+
+fd:TPFInterface a owl:Class ;
+    rdfs:subClassOf fd:FragmentInterface ;
+    rdfs:label "TPF Interface"@en ;
+    rdfs:comment "An interface to access a federation member via Triple Pattern Fragments."@en .
+
+fd:brTPFInterface a owl:Class ;
+    rdfs:subClassOf fd:FragmentInterface ;
+    rdfs:label "brTPF Interface"@en ;
+    rdfs:comment "An interface to access a federation member via Bindings-Restricted Triple Pattern Fragments."@en .
+
+fd:BoltInterface a owl:Class ;
+    rdfs:subClassOf fd:EndpointInterface ;
+    rdfs:label "Bolt Interface"@en ;
+    rdfs:comment "An interface to access a federation member using the Bolt protocol (e.g., for Neo4j)."@en .
+
+fd:GraphQLEndpointInterface a owl:Class ;
+    rdfs:subClassOf fd:EndpointInterface ;
+    rdfs:label "GraphQL Endpoint Interface"@en ;
+    rdfs:comment "An interface to access a federation member via a GraphQL API."@en .
+
+### ===== PROPERTY DECLARATIONS =====
+
+fd:interface a rdf:Property ;
+    rdfs:domain fd:FederationMember ;
+    rdfs:range  fd:EndpointInterface ;
```

Comment:
What about `fd:FragmentInterface`. It seems you need to restore the overall subclass `fd:Interface` again.